src/main/resources/openapi/sqs/vca-api-sqs-messaging-erop.yaml

```diff
@@ -267,6 +276,17 @@ components:
           example: 1f0f76a9a66f438b9bb33070
       required:
         - voterCardApplicationId
+    UpdateApplicationStatisticsMessage:
+      title: TriggerApplicationStatisticsUpdateMessage
+      type: object
+      description: SQS Message for initiating a statistics update for an application api
```

Comment:
I don't think this description is correct, how about "SQS message for initiating a statistics update for an application"